src/components/VisibleRender/VisibleRender.md

```diff
@@ -0,0 +1,30 @@
+### Examples
+
+Basic usage:
+
+```js { "props": { "data-example": "basic" } }
+const { GutterSize, TextSize } = require('../Block');
+
+const renderInView = () => {
+  return (
+    <div style={{ backgroundColor: '#adedba', transition: 'background-color 1s' }}>
+      <Block padding={GutterSize.XXLARGE} textSize={TextSize.LARGE}>In view</Block>
+    </div>
+  );
+};
+
+const renderOutOfView = (inView) => {
+  return (
+    <div style={{ backgroundColor: '#f98985', transition: 'background-color 1s' }}>
+      <Block padding={GutterSize.XXLARGE} textSize={TextSize.LARGE}>Out of view</Block>
+    </div>
+  );
+};
+
+<VisibleRender 
+  whenInView={renderInView}
+  whenOutOfView={renderOutOfView}
```

Comment:
For `ResponsiveRender` in the new client, you went with prop names `renderAbove` and `renderBelow`. Should we align with these and make these `renderInView` and `renderOutOfView`, or make the other ones `whenAbove` and `whenBelow`?

Also, should we bring `ResponsiveRender` into YamUI?

Reply:
good question. ResponsiveRender ties `react-media` to our breakpoints, do we want those breakpoints directly in YamUI? it would actually make it easier keep consistency between clients. let's do this in another PR

Reply:
and yes i'll rename these render props to include "render"